johnwayodi/store-manager
# #161166492 Add Attendant Profile view for Admin

#### What does this PR do?
Adds a view where the Admin can view the details of an individual Store Attendant
#### Description of Task to be completed?
Ensure sidebar links are properly configured to their respective views
#### How should this be manually tested?
1. `git clone https://github.com/johnwayodi/store-manager.git`
2. Ensure you are in the directory where you cloned this repository, **_store-manager_** by default
3. `git checkout ft-attendant-profile-admin-161166492`
4. open attendant-detail-admin.html on browser
#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
#161166492
#### Screenshot
![attendant-detail](https://user-images.githubusercontent.com/10683903/46839840-ab8b8180-cdc8-11e8-89c6-71fb29c1fc8e.png)

#### Questions:
Any feedback on what I can improve or change? Drop a comment or suggestion below
